docs/docs/APIdocs/layers/nn/Utilities/Pack.md

```diff
@@ -2,7 +2,7 @@
 
 **Scala:**
 ```scala
-val module = Pack(dim)
+val module = Pack(dim: Int)
```

Comment:
I think you can put this Pack to Merge Layers, which used to merge multi Tensor into one.